example/jss/covid/toy/TruncatedPoisson.bl

```diff
@@ -0,0 +1,29 @@
+package blang.distributions
+
+/** Poisson random variable. Values in \(0, 1, 2, \dots\) */
+model TruncatedPoisson {
+  random IntVar realization
+  
+  /** Mean parameter \(\lambda\). \(\lambda > 0\) */
+  param RealVar mean
+  
+  laws {
+    logf(realization, mean) {
+      if (mean <= 0) return NEGATIVE_INFINITY
+      if (realization < 3 || realization > 11) return NEGATIVE_INFINITY
+      return realization * log(mean)
```

Comment:
The way this is implemented, I don't think the sum of the probabilities will be one. There should be a renormalization to take into account the truncation.